configure

```diff
@@ -4401,6 +4460,16 @@ PLUGIN_EXTRA_DEPS =
 PLUGIN_LDFLAGS  += -shared -static-libgcc
 PRE_OBJS_FLAGS  := -Wl,-export-dynamic -Wl,-whole-archive
 POST_OBJS_FLAGS := -Wl,-no-whole-archive
+'
+		;;
+	kolibrios)
+		_plugin_suffix=".svm.dll"
+		append_var DEFINES "-DUNCACHED_PLUGINS"
+_mak_plugins='
+PLUGIN_EXTRA_DEPS      = $(EXECUTABLE)
+PLUGIN_LDFLAGS		+= -nostdlib -shared -Wl,--entry,_DllStartup -Wl,-q,--retain-symbols-file,$(srcdir)/backends/plugins/elf/plugin.syms -Wl,--enable-auto-import -shared ./libscummvm.a -ldll -lstdc++ -lsupc++ -lgcc -lc.dll
```

Comment:
As we already link scummvm with libstdc++ and so on, I suspect we will end up with several stdc++ implementations running at once which may not share their internal state.